### ISSUE TITLE
[pkgs] Improve readability of EPM console output

### ISSUE THREAD

**Issue by j-h-scheufen** [eris-ltd/eris-cli#1067]
Eris version: 0.12.0

The console output of EPM should be improved to support users better and improve readability / debugging. Especially two areas should be addressed:

## Requirements
1. Remove `abi` and `bin` from the default output.
These outputs are typically so large that it becomes extremely hard to read the jobs file as a whole or to find a particular job. These outputs should be available in the `-v` (verbose) mode.
2. Make the border between two jobs more visible.
Currently, there is no gap or anything between two jobs and the user has to scan for a line starting with `Executing Job`. There should be at least an empty line to separate the end of one job and the beginning of the next. Better even to visually make it hard to miss. Example suggestion:

Current output:
```
...
Deploying Contract                 name=TypeUtils
                                   addr=01A039A4732D3AF1CC26A1ACE4EFA42B0396984A
Executing Job                      registerTypeUtilsLibrary
```

Better:
```
...
Deploying Contract                 name=TypeUtils
                                   addr=01A039A4732D3AF1CC26A1ACE4EFA42B0396984A
#### JOB ####
# Executing Job                    registerTypeUtilsLibrary
```

**Comment by VoR0220** [comment]
https://www.youtube.com/watch?v=CRMcSAgoabw

**Comment by VoR0220** [comment]
Fixed in 0.16.0
